fetch.bs

```diff
@@ -3093,16 +3093,36 @@ steps:
 <i>authentication-fetch flag</i>.
 
 <ol>
+ <li><p>Let <var>httpRequest</var> be a new <a for=/>request</a>.
```

Comment:
It seems you should just let it be null, no? Since every substep overrides what it is.